lib/atat-web-api-stack.ts

```diff
@@ -114,6 +115,7 @@ export class AtatWebApiStack extends cdk.Stack {
       objectLockEnabled: true,
       objectLockDefaultRetention: s3.ObjectLockRetention.compliance(cdk.Duration.days(365)),
     });
+    Tags.of(accessLogsBucket).add("Classification", "UNCLASS");
```

Comment:
Are these tags expected by C5ISR? Can we create an enum class that briefly describes how/why they're used?

Reply:
This is used for STIG compliance. Not for C5ISR. this is being applied to Cloud Trail and the cloud trail s3 bucket and the load balancer access log bucket.

Reply:
working on the enum now...